### ISSUE TITLE
NoUnused.Variables: False negative for imports shadowed by another import

### ISSUE THREAD

**Issue by jfmengels** [jfmengels/elm-review-unused#70]
**Describe the bug**
Imports that are shadowed by another import are not being reported

**SSCCE (Short Self-Contained Correct Example)**
```elm
module A exposing (..)

import Accessibility exposing (Html)
import Html exposing (Html)

a : Html msg
a =
    Html.text ""
```

Create a project with both
- install `elm/html`
- Install `tesk9/accessible-html`

**Expected behavior**

The import of `Html` from `Accessibility` should be marked as unused.
